src/components/coblocks-deactivate-modal/styles/style.scss

```diff
@@ -0,0 +1,73 @@
+// WordPress variables and mixins
+@import "~@wordpress/base-styles/variables";
+@import "~@wordpress/base-styles/mixins";
+@import "~@wordpress/base-styles/breakpoints";
+@import "~@wordpress/base-styles/animations";
+@import "~@wordpress/components/src/modal/style";
+@import "~@wordpress/components/src/button-group/style";
+@import "~@wordpress/components/src/button/style";
+@import "~@wordpress/components/src/checkbox-control/style";
+@import "~@wordpress/components/src/text-control/style";
+
+.coblocks-plugin-deactivate-modal {
+	--wp-admin-theme-color: #2271b1;
```

Comment:
@sbalakrishnan1-godaddy I think if you add as a dependency `wp-components` https://github.com/godaddy-wordpress/coblocks/pull/2266/files#diff-f4655db9906d8b47edf37aae513e2191a6e714255580a417ce1ca93d8eab9336R107 you wouldn't have to do this.